app.go

```diff
@@ -603,6 +620,21 @@ func New(config ...Config) *App {
 	// Override colors
 	app.config.ColorScheme = defaultColors(app.config.ColorScheme)
 
+	// If the app is configured to use services,
+	// register a post shutdown hook to shutdown them after the server is closed.
+	if app.hasServices() {
```

Comment:
question: do you think that, for avoiding users running undesired services in production, we add here a way to check if running in production. Something like the `State` example code: https://docs.gofiber.io/next/api/state#example-environmentspecific-configuration

```
// Determine environment.
    environment := os.Getenv("ENV")
    if environment == "" {
        environment = "development"
    }
    app.State().Set("environment", environment)

    // Set environment-specific configurations.
    if environment == "development" {
        app.State().Set("apiUrl", "http://localhost:8080/api")
        app.State().Set("debug", true)
    } else {
        app.State().Set("apiUrl", "https://api.production.com")
        app.State().Set("debug", false)
    }```